openbb_platform/platform/provider/openbb_provider/standard_models/industry_pe.py

```diff
@@ -0,0 +1,26 @@
+"""Industry P/E Ratio data model."""
+
+from datetime import date as dateType
+from typing import Optional
+
+from pydantic import Field
+
+from openbb_provider.abstract.data import Data
+from openbb_provider.abstract.query_params import QueryParams
+
+
+class IndustryPeQueryParams(QueryParams):
```

Comment:
Can we have `IndustryPEData` in this file?

Reply:
Second this. It should be PE.

Reply:
Perhaps this is a stylistic choice we should define somewhere in the contributing guidelines.  If we were to make the class name a representation of the full name, `IndustryPeRatio` is more readable than `IndustryPERATIO`

Reply:
Camel case is when the first letter of each word is capitalized

ChatGPT agreed with me

<img width="519" alt="image" src="https://github.com/OpenBB-finance/OpenBBTerminal/assets/11668535/449558f7-133f-4a26-b623-9d1208eb30db">